docs/gsoc.md

```diff
@@ -20,40 +20,62 @@ If you are new to **OpenELIS Global**, we recommend starting with our [Installat
-  |Enhanced Search Functionality for OpenELIS|350 hours| This project aims to enhance the search capabilities within the OpenELIS system, providing users with a more effiecient way for retrieving vital information efficiently . We would need to Integrate a Search Engine library For indexing key coulmns and providing a faster way to Search Results |Enhanced and Efficient Search Functionality |Java , Spring |`TBD`| Mutesasira Moses|
-  |Improve Integration Tests Coverage | 350 hours| The current Service Layer Integration tests only cover a few components .This project aims to Improve Integration Tests Coverage for the critical components with in the application .Integration tests play a pivotal role in validating the interactions between various modules and ensuring the seamless functioning of the entire system | Atleast 70 % Test coverage for the Service Layer |Java , Spring ,J-Unit|`TBD`| Reagan Makoba|
-  |Improve E2E Tests Coverage|  350 hours| The Current E2E tests have a very low test coverage and are not very stable. This project is dedicated to improve the End-to-End (E2E) testing coverage for the New React front end and ensuring robust validation of the entire application workflow . E2E tests play a critical role in verifying the seamless interaction between various components, ensuring that the application functions as expected from the user's perspective | Atleast 70% Test Coverage for the new UI |react , cypress |`TBD`| Namanya Abert|
-  |Rewite Admin Page components from JSP to React|  350 hours| Most of the current Admin page Functionalities are still on the old JSP UI. This project aims at re-writing key admin components from JSP  to React including User management , Organization Management and Provider Management | Admin Functionalities migrated to the new React UI |react , TypescripT, java , Spring , REST |`TBD`| Casey Iiams-Hauser|
-  |Add Support for User UI Notifications |  175 hours| This projects aims at adding support for User Notifications in the new Front End ie when critical Orders and Results are made. This will require creating a generic Notification framework that can support adding any kind of System Notifications whenever needed | A Generic Notification Framework for Displaying user Notifications |react , Typescript, java , Spring , REST |`TBD`| Caleb Steele-Lane|
+  | Project Name                         | Project Rating | Project Description                                              | Required Skills                 | Selected Contributor | Mentors          |
```

Comment:
@Agaba-derrick Looks like you remove all the current changes and replace it with the old changes